### PR TITLE
5051 update log plot docs add support for dataframes and numpy arrays

### PR DESCRIPTION
**Pull request by mnrozhkov**
- Add DataFrame and Numpy Array in Parameters
- Add examples for DataFrame and Numpy Arra
- Add links other docs pages

Fix https://github.com/iterative/dvc.org/issues/5051

### CODE REVIEW

File: content/docs/dvclive/live/log_plot.md [hunk 1]
```diff
@@ -107,4 +108,116 @@ with Live() as live:
...
+        "sepal_array_named",
+        data,
+        x="sepal length (cm)",
+        y="sepal width (cm)",
+        template="smooth",
+        title="Numpy Array with Names"
+    )
+```
+
+The `log_plot()` method creates a `smooth` plot. Labels for `X`and `Y` extracted
```
Comment (dberenbaum) [inline content/docs/dvclive/live/log_plot.md:159]:
```suggestion
The `log_plot()` method creates a `smooth` plot. Labels for `X`and `Y`are  extracted
```

**Review by dberenbaum — APPROVED** [review]
LGTM!

**Comment by dberenbaum** [comment]
We have a few other hanging PRs in dvclive, so I would like to see if we can get them merged and release them all together. Waiting on that before merging here.

**Comment by mnrozhkov** [comment]
> We have a few other hanging PRs in dvclive, so I would like to see if we can get them merged and release them all together. Waiting on that before merging here.

Ok @dberenbaum ! Please feel free to merge when it's time! 🙌

**Pull request merged**